plugins/inputs/ipset/ipset_test.go

```diff
@@ -74,6 +74,22 @@ func TestIpset(t *testing.T) {
 				{map[string]interface{}{"packets_total": uint64(3), "bytes_total": uint64(222)}},
 			},
 		},
+		{
+			name: "Sets with and without timeouts",
+			value: `create counter-test hash:ip family inet hashsize 1024 maxelem 65536 timeout 1800 counters
+				add counter-test 192.168.1.1 timeout 1792 packets 8 bytes 672
+				create counter-test2 hash:ip family inet hashsize 1024 maxelem 65536 counters
+				add counter-test2 192.168.1.1 packets 18 bytes 673
+				`,
+			tags: []map[string]string{
+				{"set": "counter-test", "rule": "192.168.1.1"},
+				{"set": "counter-test2", "rule": "192.168.1.1"},
+			},
+			fields: [][]map[string]interface{}{
+				{map[string]interface{}{"packets_total": uint64(8), "bytes_total": uint64(672), "timeout": uint64(1792)}},
```

Comment:
Not 1800?
```suggestion
				{map[string]interface{}{"packets_total": uint64(8), "bytes_total": uint64(672), "timeout": uint64(1800)}},
```

Reply:
No. The `create` command defines the **default** value for the set and `add` command can override the value as in this case.

Reply:
Oh, indeed. Missed that.